apache/superset
# feat: Uses new table component in Drill to Detail

### SUMMARY
Uses the new table component in Drill to Detail to benefit from virtualization and deal with tables with many columns.

It also extends the new Table component with new cell and header renderers.

@eric-briscoe 

### BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF
https://user-images.githubusercontent.com/70410625/204310917-784885ee-51bd-4db9-a1e2-ad34b111f516.mov

https://user-images.githubusercontent.com/70410625/204310446-c28a3873-3231-4e21-8e57-d8a741755c2a.mov

### TESTING INSTRUCTIONS
- Check that the pagination is working as intended
- Check that the columns are correctly formatted
- Check that you can select formatting for time columns

### ADDITIONAL INFORMATION
